packages/dds/sequence/src/test/revertibles.spec.ts

```diff
@@ -22,17 +22,29 @@ import { IntervalCollection, IntervalType, SequenceInterval } from "../intervalC
 import { SharedStringFactory } from "../sequenceFactory";
 import { assertIntervals } from "./intervalUtils";
 
-describe("Sequence.Revertibles with Local Edits", () => {
+describe.only("Sequence.Revertibles with Local Edits", () => {
```

Comment:
Don't forget to remove .only here and line 186.

Reply:
fyi--we have CI validation for this at runtime in common mocha config. so shouldn't be possible to check in anything that executes a `.only`.